core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -177,6 +182,21 @@ var ContributorDashboardAdminPage = function() {
       'Submit Question Right Element taking too long to appear');
     expect(await contributionRights.getText()).toBe('Allowed');
   };
+
+  this.checkstatslistelements = async function(number) {
+    await waitFor.presenceOf(statsTable, 'stats table was not visible');
+    var statsListItems = await statsListItemsSelector();
+    expect(statsListItems.length).toBe(number);
+  };
+
+  this.checkexpandablestats = async function() {
+    await waitFor.presenceOf(statsTable, 'stats table was not visible');
```

Comment:
why precenceOf here and visibiltyOf elesewhere?

Reply:
Actually I was doing some debugging with `presenceOf` and forgot to change it to `visibiltyOf`. Changed. PTAL.